app/assets/stylesheets/pages/_product.scss

```diff
@@ -45,3 +45,24 @@ h4.sub-heading {
     margin-left: 10px;
   }
 }
+
+
+// Styling for content in tabs
+.product-tab {
+  margin-top: 15px;
+}
+
+.nav {
```

Comment:
Double check that this doesn't affect any nav styling in the navbar

Reply:
To avoid this, you could add a class on this styling like this so it's targeted properly and won't affect anything else:

.class {
  .nav {
    etc.
  }
}

Reply:
OR 

.class.nav {
}

Reply:
Class added in html - therefore, targeting .show-tabs